Support/lib/rspec/mate/switch_command.rb

```diff
@@ -140,8 +140,9 @@ def klass(relative_path, content=nil)
       def write_and_open(path)
         FileUtils.mkdir_p(File.dirname(path))
         described = described_class_for(path, base_dir)
+        is_rails = File.directory? path.sub(%r{/spec/.*$}, '/app')
```

Comment:
I’m only concerned with writing `require 'rails_helper'` if we (incorrectly) detect a Rails app. Maybe in this case we could simply check for the existence of `rails_helper.rb` and use this to decide whether to require `rails_helper` or `spec_helper`?